setup.py

```diff
@@ -85,14 +85,15 @@
 install_requires = [
     'Flask-BabelEx>=0.9.3',
     'Flask-Breadcrumbs>=0.3.0',
+    'Flask-Login>=0.4.0',
```

Comment:
Can you remove the monkey patching and provide deterministic request context with remote addr and User-Agent?